app/controllers/twilio_webhooks_controller.rb

```diff
@@ -20,6 +20,19 @@ def create_incoming_text_message
       from_phone_number: phone_number,
       client: client,
     )
+    num_media = params['NumMedia'].to_i
+
+    (0..(num_media - 1)).each do |i|
+      media_url = params["MediaUrl#{i}"]
+      content_type = params["MediaContentType#{i}"]
+      extension = MIME::Types[content_type].first.extensions.first
+      filename_with_extension = "#{media_url.split('/').last}.#{extension}"
+      contact_record.documents.attach(io: StringIO.new(Net::HTTP.get(URI(media_url))),
+                                        filename: filename_with_extension,
+                                        content_type: content_type,
+                                        identify: false)
+    end
```

Comment:
Might consider putting this in a background job to keep our responses to Twilio snappy